modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc

```diff
@@ -0,0 +1,1013 @@
+:description: Information about how to use the search-performance indexes in Neo4j.
+:test-skip: true
+= Using search-performance indexes
```

Comment:
```suggestion
= Understand the impact of indexes
```

Reply:
Going with "The impact of indexes on query performance"